app.js

```diff
@@ -2,6 +2,9 @@
 
 require('dotenv').config()
 
+const { KthAppinsights } = require('@kth/appinsights')
+KthAppinsights.init({ name: 'kurs-pm-web' })
```

Comment:
So kth's library fetches the secret "under the hood", right?

Reply:
Yes, or actually its the standard `applicationinsights` packages that is used by `@kth/appinsights` that is reading 
env variable `APPLICATIONINSIGHTS_CONNECTION_STRING` by default (and the env variable is populated by adding secretNames in ref/prod.parameters.json)

Reply:
It looks like we could pass the connection string as options instead if we want like this. It might be clearer, do you think I should change it?
```
KthAppinsights.init({ name: 'kurs-pm-web', connectionString: process.env.APPLICATIONINSIGHTS_CONNECTION_STRING })
```

Setup code in `@kth/appinsights`:
https://github.com/KTH/kth-node-appinsights/blob/2eb20b04828f88f99e51b854fdf06efcf162d5dc/src/index.ts#L37-L40

Reply:
As we've discussed. I don't think we will touch this setup very often, so let's keep it that way.